lumicks/pylake/fdcurve.py

```diff
@@ -99,6 +132,24 @@ def d(self):
             self._distance_cache = getattr(self, f"distance{self._primary_distance_channel}")
         return self._distance_cache
 
+    def get_fd_range(self, force_min=0, force_max=np.inf, distance_min=0, distance_max=np.inf):
+        """Get a subset of the fd curve.
+
+        Parameters
+        ----------
+        force_min : float
+            Minimum force. Negative values indicate that this value is defined "from end".
+        force_max : float
+            Maximum force. Negative values indicate that this value is defined "from end".
+        distance_min : float
+            Minimum force. Negative values indicate that this value is defined "from end".
+        distance_max : float
+            Maximum force. Negative values indicate that this value is defined "from end".
```

Comment:
1. The "from end" behavior is not implemented here and probably shouldn't be.
2. When you have multiple similar arg docs, you can put them on the same line:
```py
"""
...
force_min, force_max, distance_min, distance_max: float
    Force and distance limits. Some additional notes that applied to all arguments.
"""
```

Reply:
It used to be, but then I had second thoughts about it. I had forgotten to update the docstring. Amended. 👍